scripts/winesapos_convert.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Script by @GuestSneezeOSDev
+echo "Convering system to winesapOS in 3"
+sleep 1
+echo "Convering system to winesapOS in 2"
+sleep 1
+echo "Convering system to winesapOS in 1"
+sleep 1
+echo "System is converting ..."
+pacman -Sy --noconfirm
+useradd -m winesap
+echo "winesap:winesap" | chpasswd
+pacman -S git wget flatpak base-devel --noconfirm
+wget https://winesapos.lukeshort.cloud/repo/iso/winesapos-4.1.0/_test/winesapos-4.1.0-beta.1-minimal-rootfs.tar.zst
+pacman -U --noconfirm winesapos-4.1.0-beta.1-minimal-rootfs.tar.zst
+rm *.zst
+touch /etc/os-release-winesapos
+echo 'NAME="winesapOS"' >> /etc/os-release-winesapos
+echo 'PRETTY_NAME="winesapOS"' >> /etc/os-release-winesapos
+echo 'ID=winesapOS' >> /etc/os-release-winesapos
+echo 'ID_LIKE=arch' >> /etc/os-release-winesapos
+echo 'VERSION_ID=4.1.0-beta.3' >> /etc/os-release-winesapos
+echo 'HOME_URL="https://github.com/LukeShortCloud/winesapOS"' >> /etc/os-release-winesapos
+echo 'SUPPORT_URL="https://github.com/LukeShortCloud/winesapOS/issues"' >> /etc/os-release-winesapos
+echo 'BUG_REPORT_URL="https://github.com/LukeShortCloud/winesapOS/issues"' >> /etc/os-release-winesapos
+echo "[jupiter-staging]" >> /etc/pacman.conf
```

Comment:
winesapOS no longer uses the Jupiter (or Holo) repositories. These packages generally don't work on the latest upstream Arch Linux and need to be recompiled.